core/src/main/java/org/mskcc/cbio/portal/util/GlobalProperties.java

```diff
@@ -129,8 +129,6 @@
             "TCGA working groups directly.";
 
     public static final String PATIENT_VIEW_PLACEHOLDER = "patient_view_placeholder";
-    public static final String PATIENT_VIEW_GENOMIC_OVERVIEW_CNA_CUTOFF = "patient_view_genomic_overview_cna_cutoff";
-    public static final double[] DEFAULT_GENOMIC_OVERVIEW_CNA_CUTOFF = new double[]{0.2,1.5};
```

Comment:
Maybe also remove it from the example properties file? 1.5 could also be hard coded in the frontend. -- or mabye it's another pr?

Reply:
OK removed, and it looks like it's hardcoded in the frontend https://github.com/cBioPortal/cbioportal-frontend/blob/master/src/pages/patientView/genomicOverview/tracksHelper.ts#L19